test/unit/refinement-crumbs.ts

```diff
@@ -70,7 +73,7 @@ suite('RefinementCrumbs', ({ expect, spy, stub, itShouldProvideAlias }) => {
 
     it('should update field', () => {
       refinementCrumbs.previousField = 'brand';
-      refinementCrumbs.flux = { off: () => null, on: () => null } as any;
+      refinementCrumbs.flux = <any>{ off: () => null, on: () => null };
```

Comment:
Interestingly, the preferred way to cast is using `as`. I'm not asking you to revert this, though.